docs/user/user/adding-glean-to-your-project/javascript.md

```diff
@@ -0,0 +1,133 @@
+# Adding Glean to your Javascript project
+
+Javascript projects use the Javascript APIs provided by Glean.js.
+
+Currently, these bindings support collecting data from [Browser Extensions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions) (cross-browser) and [Qt/QML applications](https://doc.qt.io/qt-5/qtqml-index.html).
+
+## Requirements
+
+* Node.js >= 12.20.0.
+* npm >= 7.0.0.
+
+## Setting up the dependency
+
+The Glean.js package is distributed as an npm package [`@mozilla/glean`](https://www.npmjs.com/package/@mozilla/glean).
+
+Install Glean.js in your Javascript project, by running:
+
+```bash
+npm install @mozilla/glean
+```
+
+Then import Glean into your project:
+
+```js
+// Glean.js for webextensions
+//
+// esm
+import Glean from "@mozilla/glean/webext";
+// cjs
+const { default: Glean } = require("@mozilla/glean/webext");
+```
```

Comment:
I think it would be good to have the list of supported entry points currently available, even though it will be alist with a single item hah, it conveys the message more are coming I guess.

Reply:
Something like:

"The currently available entry points are:

- `@mozilla/glean/webext`: Glean.js for Browser Extensions"

Is it too weird that the text is in plural? 😄

Reply:
Nah, that's fine!